docker/run_setup.sh

```diff
@@ -39,19 +39,20 @@ sed "s/\"<EVM_CHAIN_ID>\"/68430/g" ./docker/genesisTemplate.json > subnetGenesis
 sed "s/\"<EVM_CHAIN_ID>\"/68431/g" ./docker/genesisTemplate.json > subnetGenesis_B.json
 sed "s/\"<EVM_CHAIN_ID>\"/68432/g" ./docker/genesisTemplate.json > subnetGenesis_C.json
 
-# Deploy three test subnets to the local network.
+# Deploy three test subnets to the local network. Note that the embedded Teleporter and Relayer support is disabled
+# because this script handles a custom deployment of each.
 echo "Creating new subnet A..."
-avalanche subnet create subneta --force --genesis ./subnetGenesis_A.json --config ./docker/defaultNodeConfig.json --evm --vm-version $SUBNET_EVM_VERSION --log-level info --skip-update-check
+avalanche subnet create subneta --force --genesis ./subnetGenesis_A.json --config ./docker/defaultNodeConfig.json --evm --vm-version $SUBNET_EVM_VERSION --log-level info --skip-update-check --teleporter=false --relayer=false
```

Comment:
could we just use the cli to deploy teleporter/relayer at this point? I'd assume there would need additional effort to update the docker setup, so not worth right now

Reply:
Yep that's the plan in the near-ish future 👍 
https://github.com/ava-labs/teleporter/pull/361#discussion_r1596857316